docs/examples/wikitext.ts

```diff
@@ -0,0 +1,52 @@
+import type { Task, models } from '@epfml/discojs-core'
+import {
+  Disco, fetchTasks, data, client as clients,
+  aggregator as aggregators, informant, serialization
+} from '@epfml/discojs-core'
+import { NodeTextLoader } from '@epfml/discojs-node'
+import fs from 'node:fs/promises'
+
+
+async function main(): Promise<void> { 
+  // Launch a server instance
+  const url = new URL('http://localhost:8080')
+  const tasks = await fetchTasks(url)
+  const task = tasks.get('wikitext-103')
+  if (task === undefined) { throw new Error('task not found') }
+  const dataset = await loadWikitextData(task)
+  const aggregator = new aggregators.MeanAggregator()
+  const client = new clients.federated.FederatedClient(url, task, aggregator)
+  const trainingInformant = new informant.FederatedInformant(task, 10)
+  const disco = new Disco(task, { scheme: 'federated', client, aggregator, informant: trainingInformant })
+  await disco.fit(dataset)
```

Comment:
it's a huge block of code which is not very understable to newcomers: can you add a few comments, new lines to make it least daunting plz?